Makefile

```
@@ -386,7 +387,41 @@ helm-chart: manifests kustomize helmify
 unit-test: ## Run unit tests for the helm chart
 	PATH="$(CURDIR)/scripts:$(PATH)" bats test/unit/
 
+##@ GKE
+
+.PHONY: create-gke
+create-gke: ## Create a new GKE cluster
```

Would it make sense to add `gcloud` as a dependency when it's used in a rule?

```suggestion
create-gke: gcloud ## Create a new GKE cluster
```

---

Hi @tvoran - gcloud CLI would be required for the other steps too, should I add them there too? might need to do if for AWS as well in a different PR

---

I wonder if we could follow the docker container `run` approach for these tools?

---

e.g.: https://hub.docker.com/r/google/cloud-sdk/ with something like: https://github.com/hashicorp/vault-secrets-operator/blob/0c7cc967d39bce5d63c22b5fbac03d488b7c1fdc/scripts/yq#L11

---

tbh, it doesn't really matter because the gcloud CLI inside the GitHub Action runner is configured via actions anyway, I've included the gcloud setup here so that if anybody from the oss community wants to try it out they can

https://github.com/hashicorp/vault-secrets-operator-ci/blob/main/.github/workflows/gke.yaml#L19

let me come back to this later, unless you guys think it's important to do it this way now?

---

What you have there is fine. We just need to ensure that it is easy for the community to make contributions, although I don't see a lot of people running these make targets outside of our CI pipeline.